modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -898,6 +932,45 @@ private void _addObjectDefinitionResources(
 		}
 	}
 
+	private void _canConnectWithRoot(
```

Comment:
I was initially going for a validate method, but i noticed the validate methods are usually on the Service layer, so i got concerned it might be a break in the convention. But i agree this is a bad name